modules/yardstick/src/main/java/org/apache/ignite/yardstick/IgniteBenchmarkArguments.java

```diff
@@ -145,6 +152,17 @@
     @Parameter(names = {"-pp", "--printPartitionStats"}, description = "Print partition statistics")
     private boolean printPartStats;
 
+    /** */
+    @Parameter(names = "--allow-operation", variableArity = true, description = "List of cache operations")
```

Comment:
Since this parameter is related only to the load test then we should rename it to "--allowedLoadTestOperations" and add a short name "ltops". Finally, update description to "List of enabled load test operations"